packages/smooth_ui_library/lib/dialogs/smooth_AlertDialog.dart

```diff
@@ -13,72 +13,99 @@ import 'package:smooth_ui_library/buttons/smooth_simple_button.dart';
 ///
 
 class SmoothAlertDialog extends StatefulWidget {
-  const SmoothAlertDialog(
+  final BuildContext context;
+  final String title;
+  final bool close;
+  final double height;
+  final Widget body;
+  final List<SmoothSimpleButton> actions;
+
+  SmoothAlertDialog(
       {Key key,
+      this.context,
```

Comment:
There should be no `BuildContext` here. Use just the `Widget build(BuildContext context)` in the `State<>` class.
Unless I missed something...